src/bun.js/node/node_os.zig

```diff
@@ -48,6 +49,24 @@ pub const Os = struct {
     pub const EOL: []const u8 = if (Environment.isWindows) "\r\n" else "\n";
     pub const devNull: []const u8 = if (Environment.isWindows) "\\\\.\nul" else "/dev/null";
 
+    pub fn availableParallelism(globalThis: *JSC.JSGlobalObject, _: *JSC.CallFrame) callconv(.C) JSC.JSValue {
+        JSC.markBinding(@src());
+	
+        return if (std.Thread.getCpuCount()) |rc| {
+            if (rc < 1) {
+                return JSC.JSValue.jsNumberFromUint64(1);
+            }
+            return JSC.JSValue.jsNumberFromUint64(rc);
+        } else |_| {
+            const err = JSC.SystemError{
+                .message = bun.String.static("Failed to get availableParallelism information"),
+                .code = bun.String.static(@as(string, @tagName(JSC.Node.ErrorCode.ERR_SYSTEM_ERROR))),
+            };
+            globalThis.vm().throwError(globalThis, err.toErrorInstance(globalThis));
```

Comment:
I don't think we should throw here. I couldn't find code in node.js that allows this to throw.

Reply:
You are right, deleted.